crates/driver/src/infra/api/routes/reveal/mod.rs

```diff
@@ -16,18 +19,20 @@ async fn route(
     state: axum::extract::State<State>,
     req: axum::Json<dto::RevealRequest>,
 ) -> Result<axum::Json<dto::RevealResponse>, (hyper::StatusCode, axum::Json<Error>)> {
+    let auction_id =
+        auction::Id::try_from(req.auction_id).map_err(Into::<api::routes::AuctionError>::into)?;
```

Comment:
Slightly less complex.
```suggestion
        auction::Id::try_from(req.auction_id).map_err(api::routes::AuctionError::from)?;
```